easybuild/easyblocks/g/gcc.py

```diff
@@ -1043,4 +1048,42 @@ def make_module_req_guess(self):
             'LD_LIBRARY_PATH': ['lib', 'lib64'],
             'MANPATH': ['man', 'share/man']
         })
+
+        # Install rpath wrappers, if built with RPATH enabled
+        if build_option('rpath'):
+            def create_rpath_wrappers(targetdir):
+                set_up_configuration(silent=True)
+                tc = Toolchain(name="system", version="0.1")
+
+                # Expect rpath_filter_dirs and rpath_include_dirs as environment variables of
+                # ,-separated lists by default. We might want to allow different separators.
+                # E.g. ":" could be used to get a similar syntax to PATH and LD_LIBRARY_PATH
```

Comment:
`:` is typically used for `_*PATH` environment variables (which specify a list of paths to search).

I think it makes sense to (only) use `,` here, since `*_DIRS` implies a list of items, and then comma-separated is reasonable.

Reply:
I've removed this for now, since this should go into the wrapper script PR in easybuild-framework. But there we can just go with a `,`-separated list